--- nix-2.3.4/misc/systemd/local.mk.orig	1970-01-01 01:00:01.000000000 +0100
+++ nix-2.3.4/misc/systemd/local.mk	2020-04-21 18:43:49.318065743 +0200
@@ -1,5 +1,5 @@
 ifeq ($(OS), Linux)
 
-  $(foreach n, nix-daemon.socket nix-daemon.service, $(eval $(call install-file-in, $(d)/$(n), $(prefix)/lib/systemd/system, 0644)))
+  $(foreach n, nix-daemon.socket nix-daemon.service, $(eval $(call install-file-in, $(d)/$(n), /lib/systemd/system, 0644)))
 
 endif
