--- nix-2.9.2/configure.ac.orig	2021-03-13 17:10:59.748579291 +0100
+++ nix-2.9.2/configure.ac	2021-03-13 19:03:22.665383108 +0100
@@ -132,7 +132,6 @@ AC_SUBST(storedir)
 AX_BOOST_BASE([1.66], [CXXFLAGS="$BOOST_CPPFLAGS $CXXFLAGS"], [AC_MSG_ERROR([Nix requires boost.])])
 # For unknown reasons, setting this directly in the ACTION-IF-FOUND above
 # ends up with LDFLAGS being empty, so we set it afterwards.
-LDFLAGS="$BOOST_LDFLAGS $LDFLAGS"
 
 # On some platforms, new-style atomics need a helper library
 AC_MSG_CHECKING(whether -latomic is needed)
